src/Generator/Base/Property.cs

```diff
@@ -7,47 +7,14 @@ internal abstract class Property : Writable
 {
     private string name = string.Empty;
 
-    private string type = string.Empty;
-
-    private bool hasBodies => Getter?.Body != null && Setter?.Body != null;
-
-    private IDictionary<string, bool> needsConvertedMapping = new Dictionary<string, bool>
-    {
-        { "int?", false },
-        { "int", false },
-        { "float", false },
-        { "float?", false },
-        { "string", false },
-        { "bool", false },
-        { "bool?", false }
-    };
-
-    private IList<string> interfaceTransformTypes = new List<string>
-    {
-        "IEnumerable",
-        "IDictionary",
-        "IList",
-        "ICollection"
-    };
-
-    internal string Type { get => type; set => HandleTypeSetter(value); }
-
-    internal string? NonInterfaceType { get; set; }
+    internal string Type { get; set; }
```

Comment:
This makes the compiler slightly more happy:
```suggestion
    internal string Type { get; set; } = string. Empty;
```
Not the biggest fan though. Haven't thought of how to get rid of the explicit `set` in Command.cs for enums yet.